destination/destination_test.go

```diff
@@ -0,0 +1,121 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package destination
+
+import (
+	"context"
+	"reflect"
+	"testing"
+	"time"
+
+	"cloud.google.com/go/pubsub"
+	"github.com/conduitio-labs/conduit-connector-gcp-pubsub/config"
+	"github.com/conduitio-labs/conduit-connector-gcp-pubsub/config/validator"
+	"github.com/conduitio-labs/conduit-connector-gcp-pubsub/models"
+)
+
+func TestDestination_Configure(t *testing.T) {
+	dest := Destination{}
+
+	tests := []struct {
+		name        string
+		in          map[string]string
+		want        Destination
+		wantErr     bool
+		expectedErr string
```

Comment:
It appears that we can do without wantErr, because `expectedErr != nil` is the same as `wantErr`

Reply:
Done.
I've deleted the `wantErr` field from all test files.
Also, I've decided to update the type of `expectedErr` to the `error` type.

[Commit 0e95bcb](https://github.com/conduitio-labs/conduit-connector-gcp-pubsub/commit/0e95bcb5d9afa64b3bb15bf0d916ab6581ba3bca)

Reply:
:+1: